alert-database/src/test/java/com/synopsys/integration/alert/database/api/DefaultCustomCertificateAccessorTest.java

```diff
@@ -0,0 +1,174 @@
+        assertFalse(customCertificateModelOptionalEmpty.isPresent());
+        CustomCertificateModel customCertificateModel = customCertificateModelOptional.get();
+        testCustomCertificateModel(expectedCustomCertificateModel, customCertificateModel);
+    }
+
+    @Test
+    public void storeCertificateTest() throws Exception {
+        CustomCertificateModel certificateModel = new CustomCertificateModel(alias, content, new Date().toString());
+
+        CustomCertificateRepository customCertificateRepository = new MockCustomCertificateRepository();
+
+        DefaultCustomCertificateAccessor customCertificateAccessor = new DefaultCustomCertificateAccessor(customCertificateRepository);
+        CustomCertificateModel customCertificateModel = customCertificateAccessor.storeCertificate(certificateModel);
+
+        testCustomCertificateModel(expectedCustomCertificateModel, customCertificateModel);
+    }
+
+    @Test
+    public void storeCertificateNullTest() throws Exception {
+        CustomCertificateRepository customCertificateRepository = new MockCustomCertificateRepository();
+        DefaultCustomCertificateAccessor customCertificateAccessor = new DefaultCustomCertificateAccessor(customCertificateRepository);
+        try {
+            customCertificateAccessor.storeCertificate(null);
+            fail();
+        } catch (AlertDatabaseConstraintException e) {
+        }
+    }
+
+    @Test
+    public void storeCertificateBlankValuesTest() throws Exception {
+        CustomCertificateRepository customCertificateRepository = new MockCustomCertificateRepository();
+        DefaultCustomCertificateAccessor customCertificateAccessor = new DefaultCustomCertificateAccessor(customCertificateRepository);
+
+        try {
+            CustomCertificateModel certificateModel = new CustomCertificateModel("", content, new Date().toString());
+            customCertificateAccessor.storeCertificate(certificateModel);
+            fail();
+        } catch (AlertDatabaseConstraintException e) {
+        }
+
+        try {
+            CustomCertificateModel certificateModel = new CustomCertificateModel(alias, "", new Date().toString());
+            customCertificateAccessor.storeCertificate(certificateModel);
+            fail();
+        } catch (AlertDatabaseConstraintException e) {
+        }
+    }
+
+    @Test
+    public void storeCertificateIdDoesNotExistTest() throws Exception {
+        CustomCertificateRepository customCertificateRepository = new MockCustomCertificateRepository();
+        DefaultCustomCertificateAccessor customCertificateAccessor = new DefaultCustomCertificateAccessor(customCertificateRepository);
+
+        try {
+            CustomCertificateModel certificateModel = new CustomCertificateModel(9L, alias, content, new Date().toString());
+            customCertificateAccessor.storeCertificate(certificateModel);
+            fail();
+        } catch (AlertDatabaseConstraintException e) {
+        }
```

Comment:
Can you add a comment in the exception block or an assertion that the exception is not null. That way other people can see that we expect that exception and it wasn't a mistake.